project.clj

```diff
@@ -12,7 +12,15 @@
                  [rm-hull/table "0.7.1"]
                  [trptcolin/versioneer "0.2.0"]
                  [org.clojure/java.classpath "1.0.0"]
-                 [org.clojure/tools.deps.alpha "0.11.910"]]
+                 [org.clojure/tools.deps.alpha "0.11.910" :exclusions [org.slf4j/jcl-over-slf4j]]
+                 [org.apache.maven.resolver/maven-resolver-transport-http "1.7.0" #_"Fixes a CVE"]
+                 [org.apache.maven/maven-core "3.8.1" #_"Fixes a CVE"]
+                 [org.eclipse.jetty/jetty-client "9.4.41.v20210516" #_"Fixes a CVE"]
```

Comment:
is this the right place for these? wont it mean we have introduced a dependency on them?

Reply:
Good question!

The dependencies themselves are not being introduced, as `lein deps :tree` reveals as of `master`.

OTOH, if hypothetically these, at some point, stopped being transitive dependencies, then yes we would be introducing them. I don't think it's a terrible worst-case scenario: `lein-nvd`/`nvd-clojure` will rarely be a production dependency in a given consumer app, so we won't be bloating those.

Ultimately, consumers can always apply :exclusions for thinning deps or resolving conflicts.

...There's the option of moving all these additions to :managed-dependencies. I think it would just work. In my personal experience `:managed-dependencies` isn't all that great though - it has corner cases, and overall it can be hard to understand (I'd say that :exclusions is simpler to understand and more used in the wild).

Reply:
As an additional thought, consumers following the advice reflected in https://github.com/rm-hull/lein-nvd#avoiding-classpath-interference will never possibly have their dev or prod dependencies affected.

So as long as this project's CI passes (i.e. tests prove that the impl is correct) there's very little impact for extra deps (if they happened someday)